.conan/test_package/CMakeLists.txt

```diff
@@ -7,20 +7,9 @@ include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
 
 find_package(wg21_linear_algebra REQUIRED)
-find_package(gtest REQUIRED)
 
 add_executable(test_linear_algebra
-        ../../linear_algebra/code/test/test_new_arithmetic.hpp
```

Comment:
This was duplicating the testing produced in the unit tests to test the generation of the Conan package.  That what fine when it was a had header-only dependencies.  However, the GTest dependency complicates and we can test the package works with the simpler test_package.cpp I've added.